core/server/src/test/java/alluxio/master/lineage/LineageMasterTest.java

```diff
@@ -67,8 +69,10 @@
   public void before() throws Exception {
     Journal journal = new ReadWriteJournal(mTestFolder.newFolder().getAbsolutePath());
     mFileSystemMaster = Mockito.mock(FileSystemMaster.class);
-    mLineageMaster =
-        new LineageMaster(new MasterContext(new MasterSource()), mFileSystemMaster, journal);
+    ThreadFactory threadPool = ThreadFactoryUtils.build("LineageMasterTest-%d", true);
+    mExecutorService = Executors.newFixedThreadPool(2, threadPool);
+    MasterContext masterContext = new MasterContext(new MasterSource());
+    mLineageMaster = new LineageMaster(masterContext, mFileSystemMaster, journal, mExecutorService);
```

Comment:
Now that we are injecting things like the executor service, does it still make sense for the `stop` method to clean up these objects?